### PR TITLE
MAGENTO2-343: fixed app only coupons during order import

### PR DESCRIPTION
**Pull request by andre-kraus**
# Pull Request Template

## Description

Shopgate specific coupons could not be added to orders.

## Type of change

- [x] Bug fix
- [ ] New feature
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I updated the CHANGELOG.md

### CODE REVIEW

File: tests/Postman/collection.json [hunk 1]
```diff
@@ -1,3214 +1,3492 @@
...
-                  "type": "text"
-                },
-                {
-                  "key": "cart[delivery_address][company]",
-                  "value": "",
-                  "type": "text"
-                },
-                {
-                  "key": "cart[delivery_address][street_1]",
-                  "value": "7 cours de larche g
```
Comment (augsteyer) [inline tests/Postman/collection.json]: may wanna remove the debug statement

Comment (augsteyer) [inline tests/Postman/collection.json]: i may not be understanding thing ,but what we testing here?

**Review by augsteyer — CHANGES_REQUESTED** [review]

File: tests/Postman/collection.json (hunk 1 above)
Comment (andre-kraus) [reply tests/Postman/collection.json]: I guess it was about testing the h20 coupon?!

Comment (augsteyer) [reply tests/Postman/collection.json]: ugh, github messed up the lines due to whitespace ignoring I had to use. I was pointing to the check_coupon that you do
```
var jsonData = pm.response.json();
if (jsonData.hasOwnProperty('items') && jsonData.items.length > 0) {
    pm.environment.set("order_item_id", jsonData.items[0].item_id);
}
```

**Review by augsteyer — APPROVED** [review]

**Pull request merged**